src/sentry/event_manager.py

```diff
@@ -703,10 +704,19 @@ def _materialize_metadata_many(jobs):
         # the point of metadata materialization as we need to ensure that
         # processing happens before.
         data = job["data"]
-        job["culprit"] = get_culprit(data)
-        job["materialized_metadata"] = metadata = materialize_metadata(data)
```

Comment:
Is `job["materialized_metadata"]` still set anywhere? Or is it not required anymore?

Reply:
it's not used anymore